cypress/integration/tests/add-match.ts

```diff
@@ -0,0 +1,44 @@
+import { AddMatchPage } from '../pages/AddMatchPage'
+import { FOOSBALL_GAME } from '../utils/data'
+import { AddPlayerPage } from '../pages/AddPlayerPage'
+import { generateRandomPlayer } from '../utils/gen'
+import { LeaderboardPage } from '../pages/LeaderboardPage'
+
+describe('Create match page', () => {
+  const addMatchPage = new AddMatchPage(FOOSBALL_GAME.name)
+  before(() => {
+    addMatchPage.visit()
+  })
+  it('renders', () => {
+    addMatchPage.locate()
+  })
+})
+
+describe('Leaderboard', () => {
+  const player1 = generateRandomPlayer()
+  const player1score = 1010
+  const player1newscore= 1026
```

Comment:
just improve the formatting pls (run it through some eslint autofix...)

Reply:
added space-infix-ops rule here https://github.com/salsita/foosball-rating/pull/158/commits/0135b3c46f018f699f43863216bf4ec49b5913a7